featuretools/__main__.py

```diff
@@ -14,9 +16,36 @@ def cli():
 def info():
     print("Featuretools version: %s" % featuretools.__version__)
     print("Featuretools installation directory: %s" % get_featuretools_root())
+    print("\nSYSTEM INFO")
+    print("-----------")
+    sys_info = get_sys_info()
+    for k, stat in sys_info:
+        print("{k}: {stat}".format(k=k, stat=stat))
 
+    print("\nINSTALLED VERSIONS")
+    print("------------------")
+    installed_packages = get_installed_packages()
+    deps = [
+        ("numpy", installed_packages['numpy']),
+        ("pandas", installed_packages['pandas']),
+        ("tqdm", installed_packages['tqdm']),
+        ("toolz", installed_packages['toolz']),
+        ("PyYAML", installed_packages['PyYAML']),
+        ("cloudpickle", installed_packages['cloudpickle']),
+        ("future", installed_packages['future']),
+        ("dask", installed_packages['dask']),
+        ("distributed", installed_packages['distributed']),
+        ("psutil", installed_packages['psutil']),
+        ("Click", installed_packages['Click']),
+        ("scikit-learn", installed_packages['scikit-learn']),
+        ("pip", installed_packages['pip']),
+        ("setuptools", installed_packages['setuptools']),
+    ]
+    for k, stat in deps:
+        print("{k}: {stat}".format(k=k, stat=stat))
 
-@click.command()
+
+@click.command(name='list-primitives')
```

Comment:
Replace the underscore with a dash to be more consistent with other CLI tools. 
https://github.com/pallets/click/issues/1123#issuecomment-425167243